proxy/src/main/java/com/velocitypowered/proxy/network/ConnectionManager.java

```diff
@@ -68,13 +69,19 @@ public void logChannelInformation() {
    */
   public void bind(final InetSocketAddress address) {
     final ServerBootstrap bootstrap = new ServerBootstrap()
-        .channel(this.transportType.serverSocketChannelClass)
+        .channelFactory(this.transportType.serverChannelFactory)
```

Comment:
I'd like some hard numbers on this. How much do we gain by using a lambda channel factory? I'm not convinced we get much of a gain.

Reply:
Going to provide tomorrow/in next 2 days

Reply:
Going to provide tomorrow/in next 2 days